examples/project_ENGIE.py

```diff
@@ -228,7 +228,7 @@ def prepare(path="data/la_haute_borne", return_value="plantdata"):
         # Build and return PlantData
         engie_plantdata = PlantData(
             analysis_type="MonteCarloAEP",  # Choosing a random type that doesn't fail validation
-            metadata="data/plant_meta.yml",
+            metadata=f"{path}/../plant_meta.yml",
```

Comment:
Not sure where my original comment went, but here it is again. This could just be the following to work consistently across OSes:
```python
metadata=path.parent / "plant_meta.yml",
```